metabase/metabase
# fix map panning and zooming is broken due to updated deps

Closes https://github.com/metabase/metabase/issues/41605

### Description

Some transitive dependency that got updated in https://github.com/metabase/metabase/pull/35118 broke pin maps panning and zooming, seems like the yarn.lock was removed and recreated. I took previously working yarn.lock and ran yarn install with new dependencies.

### How to verify

1. New question -> Sample Dataset -> People
2. Switch to map
3. Try panning and filtering via "Draw box to filter" button
4. Ensure it works
